### DISCUSSION TITLE
Regarding values property in useForm hook

### DISCUSSION THREAD

**Discussion by deepaktammali** [react-hook-form/react-hook-form#9785] [Q&A] [answered]
Hello, 

In the docs, It is mentioned that we can pass a _values_ property to the useForm to sync the form field state with external data,
But I am getting a type error when I try to pass the value.

and It also seems It is not working, If I ignore the type errors.

Am I missing something here?

`useForm({values: propsPassedAsValues})` is what I want to use.

thanks.

UPDATE:

Code sandbox re-production link: https://codesandbox.io/s/peaceful-hamilton-g2fjuz?file=/src/App.tsx

**Comment by leapful** [comment, marked as answer]
@deepaktammali 

Please provide a codesandbox url to re-produce the issue. No one can guess what is the problem with just one line of code.

**Reply by deepaktammali** [reply]
Hi,

sorry, here is the code sandbox link https://codesandbox.io/s/peaceful-hamilton-g2fjuz?file=/src/App.tsx

I am using version 7.38.0 in the reproduction.

I am trying to pass values property to useForm, this is working in the latest version (7.42.x).

**Reply by leapful** [reply]
@deepaktammali 

The option `values` has been added just after v7.41.0: https://github.com/react-hook-form/react-hook-form/releases/tag/v7.41.0

Please upgrade your react-hook-form version to latest version in order to use the new feature.

**Reply by deepaktammali** [reply]
understood @leapful thanks.
